### DISCUSSION TITLE
Minor inconsistency

### DISCUSSION THREAD

**Discussion by 0323pin** [leftwm/leftwm#975] [General]
Hi,

There's a minor inconsistency with the removal of `implicit_some`.
First of all let me make it clear that there are no issues regardless of which `theme.ron` I use.

with
```
// LeftWM theme.ron

#![enable(implicit_some)]
(
    border_width:          0,
    margin:                4,
    default_border_color:  "#282828",
    floating_border_color: "#C0C5CE",
    focused_border_color:  "#C0C5CE",
)
```

I get
```
~ > leftwm-check
:: LeftWM version: 0.4.0
:: LeftWM git hash: c46461b6c3
:: Enabled features: lefthk lefthk-core syslog-tracing sys-log
:: Checking feature dependencies . . .
    -> lefthk OK
:: Loading configuration . . .
    -> Configuration loaded OK
:: Checking keybinds . . .
    -> All keybinds OK
:: Checking environment . . .
    -> Environment OK (has XDG_RUNTIME_DIR)
:: Checking theme . . .
    -> Theme OK
```

Removing `#![enable(implicit_some)]` from `theme.ron` yeilds,
```
~ > leftwm-check
:: LeftWM version: 0.4.0
:: LeftWM git hash: c46461b6c3
:: Enabled features: lefthk lefthk-core syslog-tracing sys-log
:: Checking feature dependencies . . .
    -> lefthk OK
:: Loading configuration . . .
    -> Configuration loaded OK
:: Checking keybinds . . .
    -> All keybinds OK
:: Checking environment . . .
    -> Environment OK (has XDG_RUNTIME_DIR)
:: Checking theme . . .
ERROR: Could not parse theme file: 5:28: Expected option
```

Everything works fine, though.

**Comment by VuiMuich** [comment]
Yeah, I saw this on a recently opened issue as well. Probably missed the parser in the check to use `.with_default_extension()` method. This should become fixed with the planned `file_handler` refactor. 
Also might be, that writing the default to a new `config.ron` and migrating an old `config.toml` still puts `#![enable(implicit_some)]` at the top, I believe.

**Comment by 0323pin** [comment]
Cheers!
I'll wait for it :smile: 
It only complains about `them.ron`, `config.ron` is fine.
